lib/use-cases.typ

```diff
@@ -40,6 +40,19 @@
 	)
 }
 
+#let usecase_flows(basic: (), alt: ()) = {
```

Comment:
I would prefer `basic` and `alt` to be a part of `usecase` instead of it's own function. We simply make them optional parameters, like you've already done here.